src/config/syscheck-config.c

```diff
@@ -728,6 +728,139 @@ int read_reg(syscheck_config *syscheck, const char *entries, char **attributes,
 }
 #endif /* WIN32 */
 
+char *validate_path(char *dir) {
+    char *tmp_dir = dir;
+
+    /* Remove spaces at the beginning and the end */
+    while (*tmp_dir == ' ') {
+        tmp_dir++;
+    }
+
+    char *tmp_str = tmp_dir + strlen(tmp_dir) - 1;
+    while(*tmp_str == ' ') {
+        *tmp_str = '\0';
+        tmp_str--;
+    }
+
+    if (!strcmp(tmp_dir,"")) {
```

Comment:
Instead of using `strcmp` with an empty string is faster to check if the first character of the string is `\0`
```suggestion
    if (*tmp_dir == '\0')
```